### DISCUSSION TITLE
Valve connectors survey

### DISCUSSION THREAD

**Discussion by nliaudat** [nliaudat/esp32_8ch_motor_shield#63] [Polls]
Many users are using HmIP-VDMOT RJ11 valves. 

I removed the RJ11 connectors long time ago because you just have cut the cable and only 2 wires are needed (Green and white). 

Do you really want this type of RJ11 connector? Knowing that they greatly increase the price and size of the card

**Comment by nstadigs** [comment]
An alternative could be to only have dupont connectors (or something better) on the "main board" and then provide "adapter" boards for either RJ11 or screw terminals. That would up the cost, but ease the development of the "main board" board because you don't need to maintain multiple versions. And whenever there's a new version of the main board users can reuse their adapter board.

**Reply by nstadigs** [reply]
Something like the expansion board on this thing: https://github.com/PricelessToolkit/Droplet?tab=readme-ov-file#expansion-board-v3

**Comment by marcelveldt** [comment]
Maybe also add "no preference" to the voting ? As I honestly dont care. I have the VDMOT valves (with rj11 connector) but if needed I can remove those connectors. On the other hand with connectors makes it more plug and play.

**Comment by DerMo92** [comment]
I Like the RJ11 quick Plug and Play if you Need to change something and is need and clean.
